### DISCUSSION TITLE
The nix-shell segment in oh-my-posh in ghostty keeps showing that I'm in nix-shell

### DISCUSSION THREAD

**Discussion by Inarizxc** [ghostty-org/ghostty#3969] [Q&A] [answered]
It only happens in ghostty

**Comment by jcollie** [comment, marked as answer]
How are you launching Ghostty? If you are launching Ghostty from a Nix development environment then there are environment variables that will carry over that make oh-my-posh think that you are in a Nix shell.

**Reply by Inarizxc** [reply]
I add ghostty via flake and when I run it even with the command, it thinks I'm in nix-shell

**Comment by Inarizxc** [comment]
The thing is, in gostty outside of nix-shell, the `IN_NIX_SHELL` variable is just empty, and my configuration didn't consider such a thing
